backend/open_webui/utils/redis.py

```
@@ -18,23 +17,46 @@ def parse_redis_service_url(redis_url):
-        sentinel = redis.sentinel.Sentinel(
-            redis_sentinels,
-            port=redis_config["port"],
-            db=redis_config["db"],
-            username=redis_config["username"],
-            password=redis_config["password"],
-            decode_responses=decode_responses,
-        )
-
-        # Get a master connection from Sentinel
-        return sentinel.master_for(redis_config["service"])
+def get_redis_connection(
```

Ruff (0.11.9)

Missing return type annotation for public function `get_redis_connection`

(ANN201)